docs/layout/css-utilities.md

```diff
@@ -134,17 +134,37 @@ The float CSS property specifies that an element should be placed along the left
 ```html
 <ion-grid>
   <ion-row>
-    <ion-col class="ion-float-left">
-      <div>
-        <h3>float-left</h3>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ac vehicula lorem.
-      </div>
+    <ion-col>
+      <h3>no float</h3>
+      <img
+        alt="Silhouette of a person's head"
+        src="https://ionicframework.com/docs/img/demos/avatar.svg"
```

Comment:
Can we use the local asset?

```jsx
import useBaseUrl from '@docusaurus/useBaseUrl';

...

<img src={useBaseUrl('img/demos/avatar.svg')} />
```

That way when developing locally you get the local SVG instead of the one that is in production (in case the SVG ever changes)

Same thing for the other images.

Reply:
How would that work inside of a code block? All of the current uses like that are for actual rendered videos or images.

Reply:
🤦 Sorry I thought this was a playground by mistake. What you have now is correct